### PR TITLE
[14.0][MIG] project_purchase_link

### PR DESCRIPTION
**Pull request by Urvisha-OSI**

### CODE REVIEW

File: project_purchase_link/views/project_project_view.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                    type="object"
+                    icon="fa-pencil-square-o"
+                >
+                    <field
+                        string="Purchase Invoices"
+                        name="purchase_invoice_count"
+                        widget="statinfo"
+                    />
+                </button>
+                <button
```
Comment (CRogos) [inline project_purchase_link/views/project_project_view.xml]: I think here is something wrong. There are several buttons with the same "name" attribute, and I think some of the buttons are added twice.

**Review by CRogos — CHANGES_REQUESTED** [review]
Can you squash your FIX commits into the merge?
Maybe there is also a shorter name for the buttons, because the end is cut.
image

**Review by CRogos — APPROVED** [review]
Code Review LGTM, but squash of commits needed.

**Review by ypapouin — CHANGES_REQUESTED** [review]
Please rename this PR to « [14.0][MIG] project_purchase_link »

Functional tests are ok.

About code review:
- Add `attrs="{'invisible': [('my_value', '=', 0)]}"` to each buttons.
- Never replace existing copyright headers, please revert these changes.
- Never replace module author in manifest, please revert this change.

You have already added your name to the contributor list and its enough. In my opinion, adding your name as a module author would require a massive contribution.

**Comment by ypapouin** [comment]
Still some things, already written before :
- Please rename this PR to « [14.0][MIG] project_purchase_link » to be easily found by module name.
- Squash your [FIX] ... commits.

**Comment by rafaelbn** [comment]
@max3903 @emagdalenaC2i could you review this one or re-activate https://github.com/OCA/project/pull/773 ? Thank you!

**Comment by rafaelbn** [comment]
Please @Urvisha-OSI could you rebase and make a empty commit tu unblock runboat?

**Comment by max3903** [comment]
> Please @Urvisha-OSI could you rebase and make a empty commit tu unblock runboat?

Done

**Review by max3903 — APPROVED** [review]

**Pull request merged**